src/main/java/org/sagebionetworks/bridge/spring/controllers/HealthDataEx3Controller.java

```diff
@@ -178,10 +178,10 @@ public HealthDataRecordEx3 getRecord(@PathVariable String appId, @PathVariable S
 
     /** Retrieves the record for the given ID for self. */
     @GetMapping(path="/v3/participants/self/exporter3/healthdata/{recordId}")
-    public HealthDataRecordEx3 getRecordForSelf(@PathVariable String recordId) {
+    public HealthDataRecordEx3 getRecordForSelf(@PathVariable String recordId, @RequestParam(required = false) String download) {
         UserSession session = getAuthenticatedAndConsentedSession();
 
-        HealthDataRecordEx3 record = healthDataEx3Service.getRecord(recordId).orElseThrow(() ->
+        HealthDataRecordEx3 record = healthDataEx3Service.getRecord(recordId, "true".equalsIgnoreCase(download)).orElseThrow(() ->
```

Comment:
Add unit tests for the true and null case, to make sure we are parsing the boolean flag correctly.